File: backend-parent/qinglian-pojo/src/main/java/com/yychainsaw/dto/UserLoginDTO.java
package com.yychainsaw.dto;

import io.swagger.v3.oas.annotations.media.Schema;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
@Schema(description = "UserLoginDTO模型")
public class UserLoginDTO {
    @NotBlank(message = "用户名不能为空")
    @Schema(description = "username")
    private String username;

    @NotBlank(message = "密码不能为空")
    @Schema(description = "password")
    private String password;
}

File: QingLian-app-android/app/src/main/java/com/yychainsaw/qinglianapp/data/model/dto/UserLoginDTO.kt
package com.yychainsaw.qinglianapp.data.model.dto

import com.google.gson.annotations.SerializedName

// 对应接口文档中的 UserLoginDTO
data class UserLoginDTO(
    @SerializedName("username") val username: String,
    @SerializedName("password") val password: String
)

